frontend/src/metabase/query_builder/components/filters/pickers/RelativeDatePicker.jsx

```diff
@@ -14,6 +14,8 @@ import type {
 } from "metabase-types/types/Query";
 
 export const DATE_PERIODS: RelativeDatetimeUnit[] = [
+  "minute",
+  "hour",
```

Comment:
These are both in `TIME_PERIODS` below. I think we want to keep that distinction but instead use `ALL_PERIODS` somewhere else. (I haven't looked to see where yet)

Reply:
Ah, I see.  Seems that this should actually be controlled by the `hideTimeSelectors?` in `Props`, but I can't figure out what is (currently) setting that to `false`).

Reply:
Yeah I think that's it! I deleted `hideTimeSelectors` in `DateAllOptionsWidget.jsx` and the options appear.

Reply:
The presence of that prop there is equivalent to `hideTimeSelectors={true}`.